index.html

```diff
@@ -83,39 +84,41 @@ <h3>CANOPY</h3>
      <!-- 3rd project card -->
     <article class="project-card"> 
       <div class="project-img">
-        <img src="assets/Snapshoot-3.svg" alt="Screenshot of Project 3">
+        <img src="assets/Snapshoot Portfolio-3.svg" alt="Screenshot of Project 3">
       </div>
       <div class="project-content">
-        <h2>Tonic</h2>
+        <h2>FACEBOOK 360</h2>
         <div class="project-info">
-          <h3>CANOPY</h3>
-          <p>Back End Dev</p>
+          <h3>FACEBOOK</h3>
+          <p>Full Stack Dev</p>
           <p>2015</p>
         </div>
-        <p class="project-description">A daily selection of privately personalized reads;no accounts or sign-ups required.</p>
+        <p class="project-description">Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
         <ul class="project-tags">
           <li><a href="#">HTML</a></li> 
+          <li class="hide-on-mobile"><a href="#">Ruby on rails</a></li> 
           <li><a href="#">CSS</a></li> 
           <li><a href="#">JavaScript</a></li> 
         </ul>
         <button type="button" class="project-btn">See Project</button>
       </div>
```

Comment:
- [ ] Please add `hover` effects on the `see project` buttons on your project cards as suggested in the figma design

<img width="774" alt="Screenshot 2023-06-16 at 11 05 22 AM" src="https://github.com/CVILLA09/Portfolio/assets/44798044/fad6ce86-a2a8-4309-8b36-af8ff8122929">